python/pyspark/sql/functions.py

```diff
@@ -2301,13 +2532,46 @@ def count_distinct(col: "ColumnOrName", *cols: "ColumnOrName") -> Column:
 
     .. versionadded:: 3.2.0
 
+    Parameters
+    ----------
+    col : :class:`~pyspark.sql.Column` or str
+        first column to compute on.
+    cols : :class:`~pyspark.sql.Column` or str
+        other columns to compute on.
+
+    Returns
+    -------
+    :class:`~pyspark.sql.Column`
+        distinct values of these two column values.
+
     Examples
     --------
     >>> df.agg(count_distinct(df.age, df.name).alias('c')).collect()
     [Row(c=2)]
 
     >>> df.agg(count_distinct("age", "name").alias('c')).collect()
     [Row(c=2)]
```

Comment:
May be we can just remove the existing example since now we have a better one ?

Reply:
Removed